src/open_inwoner/configurations/tests/test_oidc.py

```diff
@@ -22,7 +24,7 @@ def setUpTestData(cls):
         openid_config.enabled = True
         openid_config.save()
 
-    def test_admin_only_enabled(self):
+    def test_admin_only_enlabled(self):
```

Comment:
*`enlabled`.. does the admin have the right label? :D

Reply:
:) Typo from the 2fa PR this was rebased on